app/templates/critter.hbs

```diff
@@ -64,6 +70,20 @@
           </tr>
         {{/each}}
       </table>
+
+      {{#if (eq model.type "fish")}}
+        <h4 class="mt-5">{{t "critter.shadowSizeImages"}}</h4>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-12 col-sm-6">
+              <img class="img-fluid" src={{concat "/assets/shadowSizeImage/high-" model.shadowSize ".png"}} alt={{t (concat "altText.shadowSizeImage.high." model.shadowSize)}}>
+            </div>
+            <div class="col-12 col-sm-6">
+              <img class="img-fluid" src={{concat "/assets/shadowSizeImage/low-" model.shadowSize ".png"}} alt={{t (concat "altText.shadowSizeImage.low." model.shadowSize)}}>
```

Comment:
Two images, one from the high camera angle and one from the low camera angle.  I play exclusively in the high angle and always find it difficult to compare the shadows when looking at a low angle screenshot.